guides/common/modules/proc_configuring-satellite-for-air-gapped-operation.adoc

```diff
@@ -0,0 +1,11 @@
+[id="configuring-satellite-for-air-gapped-operation_{context}"]
+= Configuring Satellite for Air-Gapped Operation
```

Comment:
I'm pretty sure that this title doesn't say what we need to say here. Any ideas?

Reply:
We will not really call it Satellite but {Project} right?